packages/react-examples/src/office-ui-fabric-react/__snapshots__/ColorPicker.Basic.Example.tsx.shot

```diff
@@ -614,6 +614,7 @@ exports[`Component Examples renders ColorPicker.Basic.Example.tsx correctly 1`]
                         }
                   >
                     <input
+                      aria-atomic="true"
```

Comment:
minor: it is better to write in 1 style, as written below `aria-invalid={false}`. 
Propose write like `aria-atomic={true}`

Reply:
Good point! This particular instance is auto-generated, but I can control this by using true rather than 'true' up above. Fixed in next rev